sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -188,49 +184,6 @@ def validated_config_args(args: Dict[str, str]) -> Config:
     return Config(**args)
 
 
-class ConfigureValidationStep(BaseStep):
-    """Configure validation plugin."""
-
-    def __init__(
-        self,
-        config_changes: Config,
-        client: Client,
-        manifest: Manifest,
-        tfplan: str,
-        tfvar_map: dict,
-    ):
-        super().__init__(
-            "Configure validation plugin",
-            "Changing the configuration options for tempest",
-        )
-        self.config_changes = config_changes
-        self.client = client
-        self.manifest = manifest
-        self.tfplan = tfplan
-        self.tfvar_map = tfvar_map
-
-    def run(self, status: Optional[Status] = None) -> Result:
-        """Execute step using terraform."""
-        try:
-            # See ValidationPlugin.manifest_attributes_tfvar_map
-            charms = self.tfvar_map[self.tfplan]["charms"]
-            tempest_k8s_config_var = charms["tempest-k8s"]["config"]
-            if self.config_changes.schedule is None:
-                override_tfvars = {}
-            else:
-                override_tfvars = {
-                    tempest_k8s_config_var: {"schedule": self.config_changes.schedule}
-                }
-            self.manifest.update_tfvars_and_apply_tf(
-                self.client, tfplan=self.tfplan, override_tfvars=override_tfvars
```

Comment:
The problem will be resolved by adding tfvar_config similar to [1]
Can you add this change and test instead of revert the functionality.

[1] https://github.com/canonical/snap-openstack/blob/main/sunbeam-python/sunbeam/plugins/interface/v1/openstack.py#L427